src/jit/hwintrinsiccodegenxarch.cpp

```diff
@@ -1259,24 +1257,30 @@ void CodeGen::genSSE42Intrinsic(GenTreeHWIntrinsic* node)
 }
 
 //------------------------------------------------------------------------
-// genAVXIntrinsic: Generates the code for an AVX hardware intrinsic node
+// genAvxAvx2Intrinsic: Generates the code for an AVX/AVX2 hardware intrinsic node
 //
 // Arguments:
 //    node - The hardware intrinsic node
 //
-void CodeGen::genAVXIntrinsic(GenTreeHWIntrinsic* node)
+void CodeGen::genAvxAvx2Intrinsic(GenTreeHWIntrinsic* node)
```

Comment:
Combine the importer/codegen function of AVX and AVX2 to avoid duplicating code.